app/scripts/templates/settings/clients.mustache

```diff
@@ -52,7 +52,7 @@
 
       <div class="button-row">
         <button type="submit" class="settings-button primary clients-refresh">{{#t}}Refresh{{/t}}</button>
-        <button class="settings-button secondary cancel">{{#t}}Done{{/t}}</button>
+        <button class="settings-button secondary cancel" autofocus>{{#t}}Done{{/t}}</button>
```

Comment:
Turns out that autofocus is no longer needed for that button because we only have one panel open at a time now. I removed it and all supporting code in settings panel. It should work as expected.